doc/phpactor.txt

```diff
@@ -122,10 +124,26 @@ This information is useful. Other completion mehanisms may not provide this info
 Always enable omni-completion. If another mechanism is failing to complete,
 invoke omni-complete to find out why.
 
+================================================================================
+COMPLETION PLUGINS                                *phpactor-completion-plugins*
+
+Several asynchronous completion frameworks support Phpactor
+
+                                                *phpactor-deoplete*
```

Comment:
I guess this should be aliged to the right

Reply:
Fixed.